evalml/automl/automl_search.py

```diff
@@ -619,15 +619,17 @@ def _add_baseline_pipelines(self, X, y):
         return False
 
     @staticmethod
-    def _get_mean_cv_scores_for_all_objectives(cv_data):
+    def _get_mean_cv_scores_for_all_objectives(cv_data, objective_name_to_class):
```

Comment:
Could you just call `self.objective_name_to_class` inside this function rather than pass it in as an arg? Is there a reason to choose one versus the other, since the objectives don't change post-initialization?

Reply:
Wanted to keep it static 🤷‍♂️  Yes you're right though. Functionally, there's no difference between this implementation and just adding `self` as an argument to the method.

Reply:
Got it, sounds good!